projects/lookup/src/modules/autocomplete/autocomplete.component.ts

```diff
@@ -632,6 +638,41 @@ export class SkyAutocompleteComponent
     }
   }
 
+  /**
+   * Returns the text to highlight based on exact matches, case-insensitive matches, and matches for corresponding diacritical characters (a will match à).
+   */
+  private getHighlightText(searchText: string): string[] {
+    const normalizedSearchText = SkyLookupStringUtilService.normalizeDiacritics(
+      this.searchText
+    ).toUpperCase();
```

Comment:
We might need to use `toLocaleUpperCase` instead. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase#description

Reply:
Done. Good catch.